.github/workflows/cron-update-icons.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         uses: actions/github-script@v6
         if: env.new_files == 'true'
         with:
+          github-token: ${{ secrets.PAT}}
```

Comment:
```suggestion
          github-token: ${{ secrets.PAT }}
```

Reply:
I wanted to change the commit message to the `chore` type, but I got this error and I don't know what to choose. Nothing seems to be suitable for GH actions. 🤔 😄  @DSil 
<img width="682" alt="image" src="https://github.com/kiwicom/orbit/assets/19727168/373048da-9da4-484b-a6c4-2f5633693cf4">